packages/framework/fluid-static/src/compatibilityConfiguration.ts

```diff
@@ -32,6 +32,9 @@ export const compatibilityModeRuntimeOptions: Record<
 		// Grouped batching is on by default but introduces a new type of op which is not compatible with 1.x clients.
 		enableGroupedBatching: false,
 		// TODO: Include explicit disables for things that are currently off-by-default?
+
+		// Explicitly disable running Sweep, enforcing Tombstone -- Also blocks Tombstone Autorecovery to prevent GC op
```

Comment:
I don't think this needs to say "Also blocks Tombstone Autorecovery to prevent GC op" because atorecovery is not needed if sweep is disabled.
Instead, maybe add that this will prevent sending GC op which changes doc schema and will cause 1.0 clients to fail?